backend/app/widget/static/templates/inline_code/main.gpr

```diff
@@ -5,7 +5,7 @@ project Main is
    --LANGUAGE_PLACEHOLDER--
 
    package Compiler is
-      for Switches ("ada") use ("-g", "-O0", "-gnata");
+      for Default_Switches ("ada") use ("-g", "-O0", "-gnata");
```

Comment:
"for Switches" is used for individual files.

Reply:
If Switches is passed a language instead of a filename, it acts the same as Default_Switches.